lms/services/lti_grading/factory.py

```diff
@@ -0,0 +1,13 @@
+from lms.services.lti_grading._v11 import factory as factory_v11
+from lms.services.lti_grading._v13 import factory as factory_v13
+
+
+def factory(context, request):
```

Comment:
I was having circular import problems around so now each version has it's own factory function and this one just decides which one to use.